cmd/crl-updater/main.go

```diff
@@ -81,6 +81,13 @@ type Config struct {
 		// less than the UpdatePeriod.
 		UpdateOffset config.Duration
 
+		// UpdateTimeout controls how long a single CRL shard is allowed to attempt
+		// to update before being timed out. The total CRL updating process may take
+		// significantly longer, since a full update cycle may consist of updating
+		// many shards with varying degrees of parallelism. This value must be
+		// strictly less than the UpdatePeriod. Defaults to 1 hour.
+		UpdateTimeout config.Duration `validate:"-"`
```

Comment:
We could use the config validator in addition to the check that the crl-updater performs internally.
```suggestion
		UpdateTimeout config.Duration `validate:"structonly,ltcsfield=UpdatePeriod"`
```

I chose `structonly` instead of `omitempty` because a `config.Duration` throws this error if `updateTimeout` is absent from the config file:
```
root@c7d77ec66cdc:/boulder# ./bin/crl-updater -config test/config-next/crl-updater.json
Error validating config file "test/config-next/crl-updater.json" for command "crl-updater": Key: 'Config.CRLUpdater.UpdateTimeout.Duration' Error:Field validation for 'Duration' failed on the 'required' tag
```